ui/imports/utils/Global.qml

```diff
@@ -38,8 +38,8 @@ QtObject {
     signal openImagePopup(var image, string url)
     signal openProfilePopupRequested(string publicKey, var parentPopup, var cb)
     signal openActivityCenterPopupRequested()
-    signal openSendIDRequestPopup(string publicKey, var cb)
-    signal openContactRequestPopup(string publicKey, var cb)
+    signal openSendIDRequestPopup(string publicKey, var contactDetails, var cb)
```

Comment:
I don't want to open a pandora's box, but what if contact details get changed?

Reply:
Yeah good point, some of those dialogs already handle that, some don't. My (ultimate) plan is to unify that in the common dialog. Here the reason for passing `var contactDetails` thru the signal is simply saving the call when we already have the info